### ISSUE TITLE
TS should show an error when defining only a getter in subclass but assigning in super class

### ISSUE THREAD

**Issue by bpasero** [microsoft/TypeScript#36346]
**TypeScript Version:**  3.7.2

**Code**

```ts
class Foo {
    constructor(protected readonly something: string) {

    }
}

class Bar extends Foo {
    constructor() {
        super("something");
    }

    get something(): string {
        return "other";
    }
}

new Bar();
```

**Expected behavior:**
I would like to see a compile error.

**Actual behavior:**
There is no compile error. But the code will run with this error:

```
VM27:4 Uncaught TypeError: Cannot set property something of #<Bar> which has only a getter
    at new Foo (eval at <anonymous> (main-3.js:1239), <anonymous>:4:24)
    at new Bar (eval at <anonymous> (main-3.js:1239), <anonymous>:9:9)
    at eval (eval at <anonymous> (main-3.js:1239), <anonymous>:15:1)
    at main-3.js:1239
```

**Playground Link:**

**Comment by RyanCavanaugh** [comment]
This has been fixed in 4.0
